docs/content/docs/messages-router.md

```diff
@@ -75,32 +71,36 @@ To run the Router, you need to call `Run()`.
 
 #### Ensuring that the Router is running
 
-You might sometimes want to do something after the router starts. You can use the `Running()` method for this.
+It can be useful to know if the router is running. You can use the `Running()` method for this.
 
 {{% render-md %}}
 {{% load-snippet-partial file="content/src-link/message/router.go" first_line_contains="// Running" last_line_contains="func (r *Router) Running()" padding_after="0" %}}
 {{% /render-md %}}
 
-### Execution model
+### Execution models
 
-Some *Consumers* may support only a single stream of messages - this means that until a `msg.Ack()` is sent, you will not receive any more messages.
+*Subscribers* can consume either one message at a time or multiple messages in parallel.
 
-However, some *Consumers* support multiple streams. For example, by subscribing to multiple partitions in parallel,
-multiple messages will be sent, even previous that were not Acked (e.g. the Kafka Consumer works like this).
-The router can handle this case and spawn multiple `HandlerFunc` in parallel.
+* **Single stream of messages** is the simplest approach and it means that until a `msg.Ack()` is called, the subscriber
+  will not receive any new messages.
+* **Multiple message streams** are supported only by some subscribers. By subscribing to multiple topic partitions at once,
+  several messages can be consumed in parallel, even previous messages that were not acked (for example, the Kafka subscriber
+  works like this). Router handles this model by running concurrent `HandlerFunc`s, one for each partition.
+  
+See the chosen Pub/Sub documentation for supported execution models.
```

Comment:
just simple idea: how easy it is to find it in pub/sub's docs? but it's probably for another issue...